src/main/java/com/ft/methodearticlemapper/transformation/DataTableXMLEventHandler.java

```diff
@@ -83,4 +87,30 @@ protected boolean isElementOfCorrectType(StartElement event) {
 		return false;
 	}
 
+	private List<String> createAttributesList() {
+		List<String> attributesList = new ArrayList<>();
```

Comment:
I think this could be a class field and be populated in the constructor of the class so that you recreate it every time you need to process a table.